src/utils/error.js

```diff
@@ -0,0 +1,22 @@
+const asyncErrorHandler = (func) => {
+  return async (req, res, next) => {
+    await func(req, res).catch((error) => next(error));
+  };
```

Comment:
```suggestion
  return  (req, res, next) => {
     func(req, res).catch((error) => next(error));
  };
```
더 수행할 로직이 없으므로 동기적으로 동작할 필요가 없을 것 같습니다.

Reply:
네 변경했습니다 !